source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,571 @@ Filters Builders
+      
+   * - :java-core-api:`text() <com/mongodb/client/model/Filters.html#text(java.lang.String)>`
+     - documents which contain a specified full-text search expression.
+     
+   * - :java-core-api:`where() <com/mongodb/client/model/Filters.html#where(java.lang.String)>`
+     - documents which contain a specified JavaScript expression.
+
+The following example matches documents that have a ``color`` field
+starting with the letter "p" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin regexComparison
+   :end-before: end regexComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$regex" : "^p" }
+
+The output of the above code snippet should look something like this:
+   
+.. code-block:: json
+   :copyable: false
+      
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+
+.. _bitwise:
+
+Bitwise
+-------
+
+THe bitwise operator converts a number into it's binary value to
+evaluate its bits.
+
+The bitwise operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Bitwise Method
+     - Matches
+      
+   * - :java-core-api:`bitsAllSet() <com/mongodb/client/model/Filters.html#bitsAllSet(java.lang.String,long)>`
+     - documents when the specified bits of a field are set (i.e. "1").
+
+   * - :java-core-api:`bitsAllClear() <com/mongodb/client/model/Filters.html#bitsAllClear(java.lang.String,long)>`
+     - documents when the specified bits of a field are clear (i.e. "0").
+      
+   * - :java-core-api:`bitsAnySet() <com/mongodb/client/model/Filters.html#bitsAnySet(java.lang.String,long)>`
+     - documents when at least one of the specified bits of a field are set (i.e. "1").
+     
+   * - :java-core-api:`bitsAnyClear() <com/mongodb/client/model/Filters.html#bitsAnyClear(java.lang.String,long)>`
+     - documents when at least one of the specified bits of a field are clear (i.e. "0").
+
+The following example matches documents that have a ``bitField ``field
+with bits set at positions of the corresponding bitmask "34" (i.e.
+00100010) in the ``binary_numbers`` collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin bitsComparison
+   :end-before: end bitsComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+      :copyable: false
+   
+   {  "a": { "$bitsAllSet": [1,5] } }
+
+The output of the above code snippet should look something like this:
+   
+.. code-block:: json
+   :copyable: false
+      
+   { _id: 9, a: 54, binaryValue: "00110110" }
+   { _id: 12, a: 102, binaryValue: "01100110" }
+
+.. _geospatial:
+
+Geospatial
+----------
+
+A geospatial operator evaluates a specified coordinate to a shape or location.
```

Comment:
Not sure I understand "evaluates a specified coordinate to a shape or location". Maybe you meant something like "evaluates a specified coordinate and its relation to a shape or location"?